src/snapred/backend/dao/state/CalibrantSample/Crystallography.py

```diff
@@ -24,10 +25,33 @@ class Crystallography(BaseModel):
     cif_file: str
     space_group: str
     lattice_parameters: List[float]
-    atom_type: str
-    atom_coordinates: List[float]
-    site_occupation_factor: float
-    adp: Optional[float]
+
+    class atom(BaseModel):
```

Comment:
Class names should be capitalized.  Also consider extracting this inner class out to a separate file, it will likely be used by itself in other parts of the code.

Reply:
Capitalized and separated into separate file